google/cloud/forseti/services/dao.py

```diff
@@ -1164,7 +1222,9 @@ def add_member(cls,
             if len(parents) != len(parent_type_names):
                 msg = 'parents: {}, expected: {}'.format(
                     parents, parent_type_names)
-                raise Exception('Parent not found, {}'.format(msg))
+                error_message = 'Parent not found, {}'.format(msg)
+                LOGGER.error(error_message)
+                raise Exception(error_message)
```

Comment:
I suggest logger here goes to the exception handling. So all exception get logged.